### PR TITLE
checkers/commentFormatting: exclude GoLand and VSCode custom folding regions

### PR DESCRIPTION
**Pull request by VyrCossont**
For editor documentation, see:
- https://www.jetbrains.com/help/go/code-folding-settings.html#fold-by-default-section
- https://code.visualstudio.com/docs/editor/codebasics#_folding

### CODE REVIEW

File: checkers/commentFormatting_checker.go [hunk 1]
```diff
@@ -27,12 +27,16 @@ func init() {
...
-			"//line /",        // e.g.: line /path/to/file:123
-			"//nolint ",       // e.g.: nolint
-			"//noinspection ", // e.g.: noinspection ALL, some GoLand and friends versions
-			"//export ",       // e.g.: export Foo
-			"///",             // e.g.: vertical breaker /////////////
+			"//go:generate ",   // e.g.: go:generate value
+			"//line /",         // e.g.: line /path/to/file:123
+			"//nolint ",        // e.g.: nolint
+			"//noinspection ",  // e.g.: noinspection ALL, some GoLand and friends versions
+			"//region",         // e.g.: region awawa, used by GoLand and friends for custom folding
```
Comment (cristaloleg) [inline checkers/commentFormatting_checker.go:34]: s/awawa/aware ?

Comment (VyrCossont) [reply checkers/commentFormatting_checker.go:34]: Not a typo; awawa is just a placeholder/example region name.

Comment (cristaloleg) [reply checkers/commentFormatting_checker.go:34]: Ahhh..got it.

**Review by cristaloleg — APPROVED** [review]
Great addition, thanks!

**Comment by VyrCossont** [comment]
Thank you!

**Pull request merged**